src/openscm_runner/adapters/ciceroscm_adapter/make_scenario_files.py

```diff
@@ -0,0 +1,236 @@
+"""
+Module with functionality to make emission input files
+"""
+import csv
+import logging
+import os
+
+import numpy as np
+import pandas as pd
+
+# Method to convert the unit names form tonnes to grams
+
+
+def unit_name_converter(unit):
+    """
+    Convert unit names between tonnes and grammes
+    """
+    uprefix = unit[0:2]
+    if uprefix == "Mt":
+        return "Tg{}".format(unit[2:])
+    if uprefix == "kt":
+        return "Gg{}".format(unit[2:])
+    if uprefix == "Gt":
+        return "Pg{}".format(unit[2:])
+    return unit
+
+
+# Method to find unit conversion factor
```

Comment:
This comment should be turned into the docstring for `unit_conv_factor`

Reply:
This one was also more or less already included as a docstring